frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf

```diff
@@ -100,7 +100,7 @@
             <core_rt:forEach items="${vulnerabilityList}" var="vulnerability">
             result.push({
                 "DT_RowId": "${vulnerability.id}",
-                "0": "${vulnerability.intReleaseName}",
+                "0": "<sw360:out value="${vulnerability.intComponentName}"/>",
```

Comment:
IntReleaseName changed to intComponentName?

Reply:
Copy/pasting error. Fixed.